### DISCUSSION TITLE
Web UI/API

### DISCUSSION THREAD

**Discussion by dturton** [timgit/pg-boss#314] [General]
#266 In this issue there are some projects that build a Web UI. I was wondering what everyone is using to manage the queue? Any suggestions?

**Comment by tcoats** [comment]
We're planning on expanding y5labs/boss-api and y5labs/boss-control to allow a few actions like retry now, delete, etc. No explicit timeframe. PRs welcome!

**Reply by dturton** [reply]
Fantastic! I was just reviewing the API side of things. I will for sure like to contribute to it.

**Reply by matthiasfeist** [reply]
@tcoats I just found this discussion :D how has this progressed since 2022 when you posted it? Anything new to share?

**Reply by tcoats** [reply]
@matthiasfeist Our team prefer to manage boss jobs using a PostgreSQL GUI. We'd like to build a dashboard to display the status of all our boss instances / jobs using the API, but it's in the deep backlog at this stage.

**Reply by matthiasfeist** [reply]
Understood :) Makes sense
